### ISSUE TITLE
Error: Cannot convert argument of type class java.util.HashMap

### ISSUE THREAD

**Issue by noway** [syanbo/react-native-agora#213]
```
Error: Cannot convert argument of type class java.util.HashMap
  at getUserInfoByUid(node_modules/react-native/Libraries/BatchedBridge/NativeModules.js:103:55)
  at call(src/actions/agoraState.js:162:40)
  at tryCatch(node_modules/regenerator-runtime/runtime.js:45:40)
  at call(node_modules/regenerator-runtime/runtime.js:271:22)
  at tryCatch(node_modules/regenerator-runtime/runtime.js:45:40)
  at invoke(node_modules/regenerator-runtime/runtime.js:135:20)
  at fn(node_modules/regenerator-runtime/runtime.js:170:11)
  at tryCallTwo(node_modules/promise/setimmediate/core.js:45:5)
  at doResolve(node_modules/promise/setimmediate/core.js:200:13)
  at Promise(node_modules/promise/setimmediate/core.js:66:3)
  at callInvokeWithMethodAndArg(node_modules/regenerator-runtime/runtime.js:169:20)
  at next(node_modules/regenerator-runtime/runtime.js:192:13)
  at async(node_modules/regenerator-runtime/runtime.js:216:14)
  at ? (src/actions/agoraState.js:161:80)
  at map(app:///[native code])
  at call(src/actions/agoraState.js:161:76)
  at tryCatch(node_modules/regenerator-runtime/runtime.js:45:40)
  at call(node_modules/regenerator-runtime/runtime.js:271:22)
  at tryCatch(node_modules/regenerator-runtime/runtime.js:45:40)
  at invoke(node_modules/regenerator-runtime/runtime.js:135:20)
  at fn(node_modules/regenerator-runtime/runtime.js:170:11)
  at tryCallTwo(node_modules/promise/setimmediate/core.js:45:5)
  at doResolve(node_modules/promise/setimmediate/core.js:200:13)
  at Promise(node_modules/promise/setimmediate/core.js:66:3)
  at callInvokeWithMethodAndArg(node_modules/regenerator-runtime/runtime.js:169:20)
  at next(node_modules/regenerator-runtime/runtime.js:192:13)
  at async(node_modules/regenerator-runtime/runtime.js:216:14)
  at apply(src/actions/agoraState.js:132:52)
  at apply(node_modules/react-native-agora/lib/RtcEngine.native.js:64:25)
  at apply(node_modules/react-native/Libraries/vendor/emitter/EventEmitter.js:189:33)
  at __callFunction(node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:425:42)
  at fn(node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:112:12)
  at __guard(node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:373:9)
  at value(node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:111:10)
  at value(app:///[native code])
```

![image](https://user-images.githubusercontent.com/2031472/84419820-41964000-ac6d-11ea-8c34-577cca5320f3.png)

Not sure about severity of the issue. I think since it's thrown in javascript it doesn't crash the app. Only seems to happen on Android.

react native info:

```
info Fetching system and libraries information...
System:
    OS: macOS 10.15.5
    CPU: (8) x64 Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
    Memory: 151.50 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.16.1 - ~/.nvm/versions/node/v12.16.1/bin/node
    Yarn: 1.19.2 - /usr/local/bin/yarn
    npm: 6.14.5 - ~/.nvm/versions/node/v12.16.1/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.8.4 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 13.5, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK: Not Found
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6308749
    Xcode: 11.5/11E608c - /usr/bin/xcodebuild
  Languages:
    Java: 12.0.2 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: ^16.11.0 => 16.11.0
    react-native: ^0.62.2 => 0.62.2
  npmGlobalPackages:
    *react-native*: Not Found
```

I'm using `dev/new` branch, commit https://github.com/syanbo/react-native-agora/commit/14fccb2b301d879aa0d1522d3d2a466c54109197

**Comment by noway** [comment]
FYI i couldn't reproduce it myself. Maybe it only happens when uid doesn't actually exist.

**Comment by LichKing-2234** [comment]
I think it happens when calling the `getUserInfoByUid`, you can try to reproduce it, I will fix it later.

**Comment by LichKing-2234** [comment]
fixed int 3.0.1-beta.2
